superset/assets/src/modules/colors.js

```diff
@@ -525,6 +525,16 @@ export const spectrums = {
   ],
 };
 
+export function hexToRGB(hex, alpha = 255) {
```

Comment:
@hughhhh already imports `d3-colors` . Perhaps can use that?

Reply:
Oh ere I only move the code from lower in the file since my linter was complaining about "referenced before defined". Though I don't want to sign up for it in the scope of this, PR, I totally agree that we should leverage d3-color as much as possible.

Reply:
Got it. I am fine with leaving it for later refactor outside of this PR.

Reply:
Yes actually I'm realizing that there's a need here for replacing whites with transparency for maps. That never was an issue with white backgrounds, but these maps would look better with transparency in place of whites. This calls for more of a refactor of these color gradients. We're also thinking about improving the control itself, allowing for user choosing colors and bounds, and maybe something about quantiles/ranking.